modules/core/perf/perf_umat.cpp

```diff
@@ -13,7 +13,6 @@ using namespace ::testing;
 using std::tr1::tuple;
 using std::tr1::get;
 
-namespace {
```

Comment:
In this specific case after `OCL_PERF_TEST_P` macro expansion unused variable ( opencv/modules/ts/include/opencv2/ts/ts_gtest.h:17036 file, INSTANTIATE_TEST_CASE_P macro, int variable with _dummy_ suffix) caused a warning.